packages/notification-center/src/i18n/languages/ca.ts

```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const CA: ITranslationEntry = {
+  translations: {
+    notifications: 'Notificacions',
```

Comment:
Could you add the Settings translation?
```suggestion
    settings: 'Configuració',
```

Reply:
done!